## UCSC Genome Browser Visualizations
Track data hubs can be used to visualize genomics signal tracks using the UCSC Genome Browser (for more information, please see the [Raney et al., 2014](https://academic.oup.com/bioinformatics/article/30/7/1003/232409) publication.

We developed a track data hub to visualize maxATAC-derived _in-silico_ ChIP-seq datasets in tandem with chromatin accessibility data. Data can be explored in multiple ways using this tool. 

### Visualization of Genomic Regions
After loading the track data hub (link [here]) in "My Data" > "Track Hubs" on the [UCSC Genome Browser website (https://genome.ucsc.edu/index.html)], you can select specific genomic locations to visualize all subpopulation-resolved maxATAC predictions and chromatin accessibility information in that area. The image below shows the promoter and gene body of :

### Identification of Specific TFBS
Right-clicking on the maxATAC prediction track (as demonstrated below) expands the composite track such that all TFs with binding sites in that region can be visualized.

### Visualization of Specific Subpopulations
If you would like to visualize maxATAC predictions for a subset of subpopulations (e.g., the Th1 resting and active subpopulations), click on the... and select the desired subopoulations to visualize.
